jython/DispatcherSystem/EditRoutePanel.py

```diff
@@ -985,10 +985,6 @@ def find_row_first_location(self):
                 for routelocation in self.route.getLocationsBySequenceList() \
                         if ".py" not in routelocation.getName()]
 
-        # [i for i,x in enumerate(testlist) if x == 1]
-
-        # ["foo", "bar", "baz"].index("bar")
-
```

Comment:
@Pugwash1 probably asked about these lines. They are commented out and does not do anything.